Scripts/Runtime/Configuration/WitConfiguration.cs

```diff
@@ -35,7 +35,7 @@ public class WitConfiguration : ScriptableObject
         #if UNITY_EDITOR
         public void UpdateData(Action onUpdateComplete = null)
         {
-            if (!string.IsNullOrEmpty(WitAuthUtility.AppServerToken))
+            if (!string.IsNullOrEmpty(WitAuthUtility.AppServerToken) && WitAuthUtility.AppServerToken.Length == 32)
```

Comment:
Also just calling this out, this is a second behavior change that technically falls into the same boat as the above UI change, though this is additionally saying "don't make server requests if our server token isn't a real token", which could be valuable other ways too.

Reply:
You should replace this with WithAuthUtility.IsValidToken. We have one for ServerToken, we should add one for AppServerToken so the logic is all in one place